core/variant/variant_parser.cpp

```diff
@@ -697,12 +697,12 @@ Error VariantParser::parse_value(Token &token, Variant &value, Stream *p_stream,
 			value = false;
 		} else if (id == "null" || id == "nil") {
 			value = Variant();
-		} else if (id == "inf") {
+		} else if (id == "INF" || id == "inf") {
+			// Lower-case inf, -inf, inf_neg, and nan kept for compatibility.
```

Comment:
When reading the comment does make sense, but with all of the nested conditions, not so much.
Either the comment is taken out of this sequence (similar to what you've done in `stor_fix`), or it is added for every condition individually like so:
```suggestion
			// Lowercase inf is kept for compatibility.
```